codegen.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright © 2021 Genome Research Ltd. All rights reserved.
+# Copyright © 2022 Genome Research Ltd. All rights reserved.
```

Comment:
I've applied the change. The `codegen.py` script won't add a year to the generated file, just replace it. I'm not sure if that matters since the code generated is made 'from scratch' every time? I think making add years onto the generated copyright notice might make the script more complicated than we need it to be

Reply:
That sounds reasonable for generated code.